app/boards/shields/reviung34/boards/nice_nano_v2.overlay

```diff
@@ -0,0 +1,48 @@
+#include <dt-bindings/led/led.h>
+
+&pinctrl {
+    spi3_default: spi3_default {
+        group1 {
+            psels = <NRF_PSEL(SPIM_MOSI, 0, 6)>;
+        };
+    };
+
+    spi3_sleep: spi3_sleep {
+        group1 {
+            psels = <NRF_PSEL(SPIM_MOSI, 0, 6)>;
+            low-power-enable;
+        };
+    };
+};
+
+&spi3 {
```

Comment:
Tiny nit, 4 space indents would be better in this node for consistency.

Reply:
Good catch, fixed!